fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryPlanWithIndex.java

```diff
@@ -63,6 +65,14 @@ public interface RecordQueryPlanWithIndex extends RecordQueryPlan {
     @Nonnull
     IndexScanType getScanType();
 
+    default ScanComparisons getComparisons() {
```

Comment:
The existing `getComparisons` is somewhat deprecated as the scan argument has been generalized and the ordering properties are being extracted elsewhere. So I am hesitant to see it brought back here.

Reply:
Removed `getComparisons` as it is no longer necessary: Using the `scanParameters.bind` to create the `IndexScanBounds`.